### PR TITLE
Process > Revise

### PR DESCRIPTION
**Pull request by joshuacc**
Added a new section on revisions. If anyone is available to give it a review in the next day or so, that would be awesome. 😁

### CODE REVIEW

File: manuscript/process.md [hunk 1]
```diff
@@ -387,3 +387,65 @@ then continued,
...
+correcting each problem.
+If you get stuck on something,
+leave it to the side for now.
+You'll come back to it later.
+
+Once you've completed all the corrections you were able to make,
+take another look at any you left behind.
+You may know how to fix them now.
+If so, go ahead.
+If not, we'll leave them for the next stage of the writing process: feedback.
```
Comment (jgonggrijp) [inline manuscript/process.md:451]: I like the irony here.

**Review by jgonggrijp — APPROVED** [review]
I find it well-written, clear and concise. The inspiration from programming is very clear, though it seems to me that somebody without a programming background should still be able to follow along without a problem. I also agree. I only read the diff, though, so I can't judge how it fits in with the rest of the text.

**Comment by joshuacc** [comment]
Thanks for the review @jgonggrijp !

**Comment by jgonggrijp** [comment]
Welcome!

**Pull request merged**